content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -53,20 +74,27 @@ There are two types of filters:
 - **Custom filters** (highlighted in purple above): Filter commits by one or
   more of the following fields:
 
-  - Git related fields such as Git branch, commit message, tag and author
   - Column values (values of metrics, hyperparameters, etc.) and their deltas
+  - Git related fields such as Git branch, commit message, tag and author
+
+    <admon type="info">
 
-  <admon type="info">
+    The `Branch`filter displays only the specified branch and its commits.
 
-  The `Branch`filter displays only the specified branch and its commits.
+    On the other hand, the `Commits on branch` filter will also display branches
+    that contain any commits **in common** with the specified branch. This is
```

Comment:
This sentence means that `Commits on branch = main` will also cause `feature-branch-1` and all of its commits to be shown. is this the case? or you have to do `Commits on branch = feature-branch-1` to show an merged/inactive `feature-branch-1` ?

Reply:
[Revised](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744)